accountsdb-plugin-postgres/src/accountsdb_plugin_postgres.rs

```diff
@@ -0,0 +1,348 @@
+                    Err(err) => {
+                        return Err(AccountsDbPluginError::Custom(
+                            Box::new(AccountsDbPluginPostgresError::DataStoreConnectionError {
+                                msg: format!(
+                                "Error in connecting to the PostgreSQL database: {:?} host: {:?} user: {:?} config: {:?}",
+                                err, config.host, config.user, connection_str),
+                            })));
+                    }
+                    Ok(mut client) => {
+                        let result = client.prepare("INSERT INTO account (pubkey, slot, owner, lamports, executable, rent_epoch, data, updated_on) \
+                            VALUES ($1, $2, $3, $4, $5, $6, $7, $8) \
+                            ON CONFLICT (pubkey) DO UPDATE SET slot=$2, owner=$3, lamports=$4, executable=$5, rent_epoch=$6, \
+                            data=$7, updated_on=$8");
+
+                        match result {
+                            Err(err) => {
+                                return Err(AccountsDbPluginError::Custom(
+                                    Box::new(AccountsDbPluginPostgresError::DataSchemaError {
+                                    msg: format!(
+                                        "Error in preparing for the accounts update PostgreSQL database: {:?} host: {:?} user: {:?} config: {:?}",
+                                        err, config.host, config.user, connection_str
+                                    ),
+                                })));
+                            }
+                            Ok(update_account_stmt) => {
+                                self.client = Some(Mutex::new(PostgresSqlClientWrapper {
+                                    client,
+                                    update_account_stmt,
+                                }));
+                            }
+                        }
+                    }
+                }
+            }
+        }
+
+        Ok(())
+    }
+
+    /// Unload all plugins and loaded plugin libraries, making sure to fire
+    /// their `on_plugin_unload()` methods so they can do any necessary cleanup.
+    fn on_unload(&mut self) {
+        info!("Unloading plugin: {:?}", self.name());
+    }
+
+    fn update_account(&mut self, account: ReplicaAccountInfoVersions, slot: u64) -> Result<()> {
+        match account {
+            ReplicaAccountInfoVersions::V0_0_1(account) => {
+                if let Some(accounts_selector) = &self.accounts_selector {
+                    if !accounts_selector.is_account_selected(account.pubkey, account.owner) {
+                        return Ok(());
+                    }
+                } else {
+                    return Ok(());
+                }
+
+                debug!(
+                    "Updating account {:?} {:?} at slot {:?} using account selector {:?}",
+                    account.pubkey,
+                    account.owner,
```

Comment:
I think we need to stringify these fields somewhere before here, don't we?
We want to print pretty strings, plus your database schema is expecting varchar

Reply:
The database is not using VARCHAR any longer for efficiency.

Reply:
I changed for the debug message.

Reply:
I see VARCHAR in the schema sql...? https://github.com/solana-labs/solana/pull/20047/files#diff-27e23ad1496611c1e34cb0a0349fc572ab7dafcd9e9bbe4b8ae5d7c20b0c2e9c

Reply:
Good catch -- I missed cherry-picking the schema change. Updated